test/performance/search/dream_index/interleaved_bloom_filter_benchmark.cpp

```diff
@@ -63,6 +70,52 @@ void emplace_benchmark(::benchmark::State & state)
     state.counters["hashes/sec"] = hashes_per_second(std::ranges::size(hash_values));
 }
 
+template <typename ibf_type>
+void clear_benchmark(::benchmark::State & state)
+{
+    auto && [ bin_indices, hash_values, ibf ] = set_up<ibf_type>(state.range(0),
+                                                                 state.range(1),
+                                                                 state.range(2),
+                                                                 state.range(3));
+    (void) bin_indices;
+    (void) hash_values;
+
+    std::vector<seqan3::bin_index> bin_range = std::views::iota(0u, static_cast<size_t>(state.range(0)))
+                                             | std::views::transform([] (size_t i) { return seqan3::bin_index{i}; })
+                                             | seqan3::views::to<std::vector>;
+
+    for (auto _ : state)
+    {
+        for (auto bin : bin_range)
+            ibf.clear(bin);
+    }
+
+    state.counters["bins/sec"] = hashes_per_second(std::ranges::size(bin_range));
+}
+
+template <typename ibf_type>
+void clear_range_benchmark(::benchmark::State & state)
+{
+    auto && [ bin_indices, hash_values, ibf ] = set_up<ibf_type>(state.range(0),
+                                                                 state.range(1),
+                                                                 state.range(2),
+                                                                 state.range(3));
+    (void) bin_indices;
+    (void) hash_values;
+
+    std::vector<seqan3::bin_index> bin_range = std::views::iota(0u, static_cast<size_t>(state.range(0)))
```

Comment:
Maybe I am blind, but is this not the same as in clear_benchmark()? I don't see what this function does different.

Reply:
This expression is the same.
Inside the benchmarked loop we use either the clear on a single bin or the clear on a range.